documentation/src/main/wiki/Home.md

```diff
@@ -4,20 +4,15 @@
   <img src="https://github.com/Cognifide/knotx/blob/master/icons/180x180.png?raw=true" alt="Knot.x Logo"/>
 </p>
 
-A simplified description of **Knot.x** can be `a tool which converts a static page (template) into a 
-dynamic page driven by the data provided by external sources`
-
-In short, we call **Knot.x** a **reactive multisource assembler**.
+**Knot.x** is a high-efficient and scalable integration platform for modern websites written in [Vert.x](http://vertx.io/) technology.
```

Comment:
high-efficient -> highly-efficient
written in Vert.x technology -> developed in Java on top of Vert.x framework